src/components/Sidebar.tsx

```diff
@@ -0,0 +1,48 @@
+import { Link, useLocation } from 'react-router-dom';
+import { useApp } from '../contexts/AppContext';
+import {FiBarChart2, FiEdit, FiHome, FiUpload} from "react-icons/fi";
```

Comment:
`useApp` không tồn tại 

```suggestion
import {Link, useLocation, useNavigate} from 'react-router-dom';
import {FiBarChart2, FiEdit, FiHome} from "react-icons/fi";
```